core/builder.py

```diff
@@ -45,14 +46,18 @@ def user(
         """
         user_count = next(self._user_count)
 
-        default_props = dict(username=f"user-{user_count}")
```

Comment:
I actually intended each user to have a 'test friendly' username. The real tool would never use firstname.lastname as a username, so no need to do so in tests either?

Reply:
I can just use email as username instead, if that solves the issue @kws ?